charts/tempo/Chart.yaml

```
@@ -3,7 +3,7 @@ name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
 version: 1.7.1
```

`version` should be bumped too.

```suggestion
version: 1.7.2
```